extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -154,7 +158,9 @@ private CatalogMetadata addPropertiesToCatalog(JSONObject catalog, String uuid_c
             catalogMetadata.setDescription(FACTORY.createLiteral(catalog.getString(DESCRIPTIONPREDICATE)));
         }
         if (catalog.has(LANGUAGEPREDICATE)) {
-            catalogMetadata.setLanguage(FACTORY.createIRI(catalog.getString(LANGUAGEPREDICATE)));
+            if (!catalog.getString(LANGUAGEPREDICATE).equals("[\"\"]")) {
```

Comment:
I assume this is checking for an empty json array? Does the `JSONObject` have other methods for that?